client/state/sites/selectors.js

```diff
@@ -73,6 +73,38 @@ export const getSite = createSelector(
 );
 
 /**
+ * Returns a the ID of the primary site.
+ *
+ * @param  {Object}   state  Global state tree
+ * @return {?Number}         The ID of the primary site
+ */
+export function getPrimarySiteId( state ) {
+	const site = find( state.sites.items, 'is_primary' );
```

Comment:
Where does `is_primary` come from? Before, we had to assign an attribute of `primary` in sites-list. ee previous attempt https://github.com/Automattic/wp-calypso/pull/4819